client-internal/src/main/java/org/terracotta/angela/client/config/TsaConfigurationContext.java

```diff
@@ -31,8 +31,6 @@
   TerracottaCommandLineEnvironment getTerracottaCommandLineEnvironment(String whatFor);
 
   interface TerracottaCommandLineEnvironmentKeys {
-    String CLUSTER_TOOL = "ClusterTool";
```

Comment:
Not needed anymore as these are separate tools now. env will need to be set using the specific tool's APIs.